### PR TITLE
Fix #2126: Highfi Revision card [Tablet]

### LINKED ISSUE #2126: [Tablet] Highfi Revision Card
Currently `RevisionCard` screen does not have dedicated tablet layout design in develop.
Implement the `RevisionCard` for landscape and portrait mode for tablet devices.
Mock Link: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/a2d04ed0-de4e-40fe-8a83-824663b30d3f/

Reference: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #2126 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Portrait | Landscape 
--- | --- 
 |

### CODE REVIEW

**Comment by FareesHussain** [comment]
@rt4914 please check

**Comment by anandwana001** [comment]
@FareesHussain please add the screenshots in the description

**Comment by FareesHussain** [comment]
Portrait | Landscape 
--- | --- 
 |

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      android:layout_height="wrap_content"
+      android:gravity="center"
+      android:orientation="horizontal">
+
+      <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:minWidth="640dp"
+        android:orientation="vertical"
+        android:paddingBottom="145dp">
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: The dimensions should be in multiples of `4dp` therefore I think we should make it 148dp

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        <TextView
+          android:id="@+id/revision_card_explanation_text"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="52dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Remove start and end margin as we have already mentioned minWidth.

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp" />
+
+        <Button
+          android:id="@+id/revision_card_return_button"
+          style="@style/StateButtonActive"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:layout_marginStart="52dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Remove start and end margin as we have already mentioned minWidth.

File: app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        android:layout_height="wrap_content"
+        android:minWidth="512dp"
+        android:orientation="vertical"
+        android:paddingBottom="160dp">
+
+        <TextView
+          android:id="@+id/revision_card_explanation_text"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="28dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml]: Remove start and end margin as we have already mentioned minWidth.

File: app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp" />
+
+        <Button
+          android:id="@+id/revision_card_return_button"
+          style="@style/StateButtonActive"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:layout_marginStart="28dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml]: Remove start and end margin as we have already mentioned minWidth.

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        android:minWidth="640dp"
+        android:orientation="vertical"
+        android:paddingBottom="145dp">
+
+        <TextView
+          android:id="@+id/revision_card_explanation_text"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="52dp"
+          android:layout_marginTop="24dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: `MarginTop` is `32dp` in mocks.

Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: MarginTop is `36dp` in mocks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks @FareesHussain PTAL.

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml (hunk 6 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: @rt4914 this is for portrait right?

Comment (rt4914) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Clarification
`36dp` - portrait
`32dp` - landscape.

Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Done in portrait

Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml (hunk 5 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml (hunk 4 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml (hunk 3 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml (hunk 2 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml (hunk 1 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**